lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
People don't see the inline documentation for the `review` object, so it needs to be attached to this DSL attribute to end up on the reference - http://danger.systems/reference.html